clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/PushJobZstdConfig.java

```diff
@@ -12,17 +12,26 @@
 public class PushJobZstdConfig {
   private ZstdDictTrainer zstdDictTrainer;
   private int maxBytesPerFile;
-  private int dictSize;
-  private int sampleSize;
+  private int maxDictSize;
+  private int maxSampleSize;
   private int filledSize; // Duplicate of filledSize in ZstdDictTrainer as there is no getter for this
+  /**
+   * Known <a href="https://github.com/luben/zstd-jni/issues/253">zstd lib issue</a> which
+   * crashes if the input sample is too small. So adding a preventive check to skip training
+   * the dictionary in such cases using minNumberOfSamples. Keeping it simple and hard coding
+   * it as if this check doesn't prevent some edge cases then we can disable the feature itself
+   */
+  private int minNumberOfSamples = 20;
```

Comment:
nit: can you make it a private static constant?

Reply:
yes, done.